lib/cucumber/core/test/result.rb

```diff
@@ -374,7 +374,7 @@ def total(for_status = nil)
             if for_status
               @totals.fetch(for_status, 0)
             else
-              @totals.reduce(0) { |total, status| total += status[1] }
+              @totals.values.reduce(0) { |total,status| total + status }
```

Comment:
Small functional tweak here. Part 3 of 3

Reply:
I think `status` is the wrong name for the block param here. It's the count of scenarios with the given status, right?

```suggestion
              @totals.values.reduce(0) { |total,count| total + count }
```